public/translations/cat-CAT.json

```diff
@@ -941,4 +948,4 @@
       }
     }
   }
-}
+}
```

Comment:
Add LF file endings on your editor to avoid this

Reply:
My bad, didn't even realize that was happening